### PR TITLE
Added screenshorts and hints

### PR DESCRIPTION
**Pull request by SvitlanaDidkivska**
In several places, it seemed that for novice users text instructions only might be not enough for clear understanding.

IMO several screenshots would be a good addition to the instructions.

### CODE REVIEW

File: 02-DocumentVersionTracking/DocumentVersionTracking.md [hunk 1]
```diff
@@ -174,6 +174,8 @@ stalj@uek.krakow.pl
 
     One of the most commonly used websites for remote repository storage is Microsoft GitHub. To store data from your local FirstProject repository in a remote GitHub repository (<https://github.com/>), in a web browse log in and create a new, completely empty repository called "FirstProject" in it.
 
+    ![](screenshots/new_repository_creation_on_github.jpg)
```
Comment (stalj) [inline 02-DocumentVersionTracking/DocumentVersionTracking.md:177]: 1. It has to be public, not private, repository to control students' progress. So please modify the screen and mark the public
2. I also suggest doing some red circle/oval around the 'Repository name' field, as you did on the other screenshots

File: 02-DocumentVersionTracking/DocumentVersionTracking.md [hunk 2]
```diff
@@ -217,6 +219,10 @@ stalj@uek.krakow.pl
         ```
         git clone https://github.com/YOUR-ACCOUNT-NAME/SecondProject.git
         ```
+    
+        Hint: You may also get a permalink to your repository from a GitHub repository's page.  
+    
+        ![](screenshots/place_where_you_can_copy_your_project_HTTPS.jpg)
```
Comment (stalj) [inline 02-DocumentVersionTracking/DocumentVersionTracking.md:225]: IMO please add some red oval around:
1. The green 'Code' button
2. The http address and copy button on the right

File: 02-DocumentVersionTracking/DocumentVersionTracking.md [hunk 3]
```diff
@@ -246,6 +252,10 @@ stalj@uek.krakow.pl
 
 1. Display the the content of created repository in VSCode.
 
+Hint: You can also do it with the help of "Open Folder..." button at the welcome page.
+
+![](screenshots/Display_the_content_of_created_repository_in_VSCode.jpg)
```
Comment (stalj) [inline 02-DocumentVersionTracking/DocumentVersionTracking.md:257]: As far as I remember, the Welcome screen is swithed off in the computer labs. So you could red mark the Documents icon (first in the left panel) and create som hint on how to do with opening a folder

BTW, I have prepared a dedicated copy of VSCode with some necessary extensions for this course. In the computer labs, on the Windows desktop, you should find a VSC folder and in it, two VSCodes: one for my course in Java and one for this Data Notations course.

**Review by stalj — CHANGES_REQUESTED** [review]
I added some comments to your changes, especially to modify scrennshots.

File: 02-DocumentVersionTracking/DocumentVersionTracking.md (hunk 1 above)
Comment (SvitlanaDidkivska) [reply 02-DocumentVersionTracking/DocumentVersionTracking.md:177]: Changes submitted.

File: 02-DocumentVersionTracking/DocumentVersionTracking.md (hunk 2 above)
Comment (SvitlanaDidkivska) [reply 02-DocumentVersionTracking/DocumentVersionTracking.md:225]: Changes submitted.

File: 02-DocumentVersionTracking/DocumentVersionTracking.md (hunk 3 above)
Comment (SvitlanaDidkivska) [reply 02-DocumentVersionTracking/DocumentVersionTracking.md:257]: Changes submitted.

**Review by stalj — APPROVED** [review]

**Pull request merged**